api/buffer/design.go

```diff
@@ -358,6 +358,15 @@ var Receiver = Type("Receiver", func() {
 		Description("List of receiver exports. A receiver may have a maximum of 20 exports.")
 	})
 	Example(exampleReceiver())
+	Required("name", "url", "exports")
+})
+
+var ReceiverResponse = Type("ReceiverResponse", func() {
+	Attribute("exports", ArrayOf(ExportResponse), func() {
```

Comment:
Tady to neudělá override `Receiver.exports`. Nedohledal jsem, tak to udělat. Pořadí `Attribute` a `Extend` na to nemá vliv.

Reply:
Další možnost je smířit se s tím, že budeme mít v response optional ids

Reply:
Nevím, jestli si tohle viděl @michaljurecko, stačí, jestli tě něco nenapadne z hlavy, nemusíš to zkoumat

Reply:
pozeral som, pockaj prosim este chvilku, bolo tam viac veci co trebalo doriesit ... a `Extend` sa na niektorych miestach nedal pouzit pre bug v nasom `token` extension (takze mozes nadavat na mna a nie na `goa` :slightly_smiling_face: `, finishujem PR .